src/test/model_test.py

```diff
@@ -450,7 +454,10 @@ def test_model_encode(self):
 
         model_class = ModelSentenceTransformer(MODEL_ROOT)
         response_answer = model_class.encode("123", DATA_TEST_SINGLE_TEXT, 'deepset/roberta-base-squad2')
-        self.assertEqual(str(response_answer), answer)
+        answer = json.loads(answer)
+        result = np.abs(np.subtract(answer['result'], response_answer['result']))
+        average = sum(sum(result)) / (result.shape[0] * result.shape[1])
+        self.assertTrue(threshold > average)
```

Comment:
I like this approach @AdrianaMCarmiol. What's the average value typically? Maybe the threshold is currently to high (0.1).

Reply:
Normally the difference between the vectors ends up being around 1x10<sup>-7</sup>

Reply:
In that case I think you can make the threshold more strict, something like 0.001.